tests/tvk-oneclick/creat_virtual_cluster.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+def create_vcluster()
```

Comment:
write `cleanup` func as well, which will clean up all the resources and virtual cluster you've created on cluster as a part of CI process.
`cleanup` func should run at the end, irrespective of test job failure/success